blazor/in-place-editor/events.md

```diff
@@ -97,7 +141,7 @@ This section explains the list of events of the In-place Editor's component whic
 
 ## ValueChange
 
-`ValueChange` event triggers when the integrated component value has changed that render based on the `type` property in the In-place editor.
+[ValueChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.InPlaceEditorEvents-1.html#Syncfusion_Blazor_InPlaceEditor_InPlaceEditorEvents_1_ValueChange) event triggers when the integrated component value has changed that render based on the [type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.SfInPlaceEditor-1.html#Syncfusion_Blazor_InPlaceEditor_SfInPlaceEditor_1_Type) property in the In-place editor.
```

Comment:
The [ValueChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.InPlaceEditorEvents-1.html#Syncfusion_Blazor_InPlaceEditor_InPlaceEditorEvents_1_ValueChange) event triggers when the integrated component value has changed that render based on the [type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.InPlaceEditor.SfInPlaceEditor-1.html#Syncfusion_Blazor_InPlaceEditor_SfInPlaceEditor_1_Type) property in the In-place editor.